src/app/pages/ArticlePage/index.test.tsx

```diff
@@ -113,8 +114,7 @@ const Context = ({
 beforeEach(() => {
   process.env.SIMORGH_ICHEF_BASE_URL = 'https://ichef.test.bbci.co.uk';
 
-  fetch.resetMocks();
-  ATIAnalytics.mockImplementation(
+  (ATIAnalytics as jest.Mock).mockImplementation(
```

Comment:
Will these mocks need to be reset in a `beforeEach` or `afterEach`?

Reply:
Tests further down mock the implementation again https://github.com/bbc/simorgh/blob/afdc1fb70016ab77ebe95be8f3a7381f435413e1/src/app/pages/ArticlePage/index.test.tsx#L798 so I think its probably ok to not reset?

Reply:
Hmm. What if we were to reset it in an `afterAll` which means that this test cleans up after itself?